gem/styles/gem-springster/modules/tags/_m-tag-cloud.scss

```diff
@@ -64,17 +64,17 @@
 }
 
 // //Tag Cloud for Article Page, dont see the need of this?
```

Comment:
@projectmushroom this style tag is for article tag variation e.g. 
<img width="368" alt="screenshot 2017-05-08 13 08 56" src="https://cloud.githubusercontent.com/assets/1453355/25802816/5f0530d2-33f4-11e7-884a-c3ff9540df41.png">